KubeArmor/feeder/feeder.go

```diff
@@ -32,6 +34,16 @@ import (
 
 // Running flag
 var Running bool
+var CACertPath = cert.CertPath{
+	Base:     cfg.GlobalCfg.TLSCertPath,
+	CertFile: "ca.crt",
+	KeyFile:  "ca.key",
+}
+var ServerCertPath = cert.CertPath{
+	Base:     cfg.GlobalCfg.TLSCertPath,
+	CertFile: "server.crt",
+	KeyFile:  "server.key",
+}
```

Comment:
Why are these global variabled? and cannot be used from GlobalCfg directly?

Reply:
the path to cert (directory) is configurable and can be specified using `--tlsCertPath` flag, and for simplicity we assume that ca/client/server certs are exist with particular file names. i.e. `ca.crt`, `ca.key`, `server.crt` etc.

Reply:
But if we make it a global variable. It's never going to respect the value right? It's only going to use the build time value i.e. whatever is the default value is?

Reply:
It set the path to --tlsCertPath=<cert path> runtime flag value, currently the dynamic reload of any tls config is not being supported it require to restart kubearmor process.

Reply:
I tried the following code
I set a global variable to another global variable
It only retained the build time value
Here's the code
```go
package main

import (
	"flag"
	"fmt"
)

var svar string = "original value"
var Globalvariable string = svar

func main() {
	flag.StringVar(&svar, "svar", "bar", "a string var")

	flag.Parse()

	fmt.Println("svar:", svar)
	fmt.Println("globalvar:", Globalvariable)
}
```

Here's the behaviour
![image](https://github.com/kubearmor/KubeArmor/assets/47106543/84b92075-2c83-4441-9cb2-aa0fb6336fc7)

Am I missing something here?

Reply:
Got your point, yes you're right it will set the default value only. my bad i will update it. thanks for pointing this out :microscope:

Reply:
is it okay if we hard code the file names for certs? do you think we need to have the configs for that too?